src/components/404.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+
+function PageNotFound() {
+  return (
+    <section className="page_404">
+      <div className="container">
+        <div className="four_zero_four_bg">
+          <h1 className="text-center ">404</h1>
+        </div>
+
+        <div className="contant_box_404">
+          <h3 className="h2">Looks like you are lost</h3>
+
+          <p>We dont have the page you are looking for</p>
+          <Button className="btn-one">Go home</Button>
```

Comment:
What happens when the user clicks the button? There doesn't seem to be any event triggered and this is also not a link

Reply:
Forgot to include the link to the homepage. I have made changes, users will now be redirected to home after clicking the button at the end of the page.